modules/ai/assets/js/media-library/edit-button.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { RequestIdsProvider } from '../editor/context/requests-ids';
+import styled from 'styled-components';
+import { __ } from '@wordpress/i18n';
+import { AIMedia, getImageId } from './utils';
+
+const Icon = styled.i`
+	color: #C00BB9;
+	padding-right: 0.5em;
+	cursor: pointer;
+`;
+
+const StyledButton = styled.a`
+	font: inherit;
+	display: inline-block;
+	text-decoration: none;
+	font-size: 13px;
+	line-height: 2.15384615;
+	padding: 0 10px;
+	cursor: pointer;
+	border-width: 1px;
+	border-style: solid;
+	border-radius: 3px;
+	white-space: nowrap;
+	box-sizing: border-box;
+	background: #f6f7f7;
+	color: #C00BB9;
+
+	&:hover {
+		color: #C00BB9;
+		background: #eeefef;
+	}
+`;
```

Comment:
Please make sure all custom CSS in the PR are:
- RTL ready. e.g., instead of `padding-right` use `padding-inline-end`
- Dark mode ready. e.g., instead of using the `#C00BB9` color, find it in the `scss/helpers/variables.scss` file and use the `$e-pink-900` variable.

After that please test it on an RTL language and dark mode.